tools/mhddos/start.py

```diff
@@ -0,0 +1,1194 @@
+#!/usr/bin/env python3
+import argparse
+import random
+from contextlib import suppress
+from json import load
+from logging import basicConfig, getLogger, shutdown
+from math import log2, trunc
+from multiprocessing import RawValue
+from os import urandom as randbytes
+from pathlib import Path
+from random import choice as randchoice
+from re import compile
+from socket import (
+    AF_INET,
+    IPPROTO_TCP,
+    SOCK_DGRAM,
+    SOCK_RAW,
+    SOCK_STREAM,
+    TCP_NODELAY,
+    gethostname,
+    socket,
+)
+from ssl import CERT_NONE, SSLContext, create_default_context
+from sys import argv
+from sys import exit as _exit
+from threading import Event, Thread
+from time import sleep, time
+from typing import Any, List, Set
+from urllib import parse
+
+import urllib3
+import yarl
+from certifi import where
+from cloudscraper import create_scraper
+from dns import resolver
+from icmplib import ping
+from psutil import cpu_percent, net_io_counters, process_iter, virtual_memory
+from PyRoxy import Tools as ProxyTools
+from requests import Response, Session, cookies, get
+
+basicConfig(format="[%(asctime)s - %(levelname)s] %(message)s", datefmt="%H:%M:%S")
+logger = getLogger("MHDDoS")
+logger.setLevel("INFO")
+ctx: SSLContext = create_default_context(cafile=where())
+ctx.check_hostname = False
+ctx.verify_mode = CERT_NONE
+urllib3.disable_warnings()
+
+__version__: str = "2.4 SNAPSHOT"
+__dir__: Path = Path(__file__).parent
+__ip__: Any = None
+tor2webs = [
+    "onion.city",
+    "onion.cab",
+    "onion.direct",
+    "onion.sh",
+    "onion.link",
+    "onion.ws",
+    "onion.pet",
+    "onion.rip",
+    "onion.plus",
+    "onion.top",
+    "onion.si",
+    "onion.ly",
+    "onion.my",
+    "onion.sh",
+    "onion.lu",
+    "onion.casa",
+    "onion.com.de",
+    "onion.foundation",
+    "onion.rodeo",
+    "onion.lat",
+    "tor2web.org",
+    "tor2web.fi",
+    "tor2web.blutmagie.de",
+    "tor2web.to",
+    "tor2web.io",
+    "tor2web.in",
+    "tor2web.it",
+    "tor2web.xyz",
+    "tor2web.su",
+    "darknet.to",
+    "s1.tor-gateways.de",
+    "s2.tor-gateways.de",
+    "s3.tor-gateways.de",
+    "s4.tor-gateways.de",
+    "s5.tor-gateways.de",
+]
+
+with open(__dir__ / "config.json") as f:
+    con = load(f)
```

Comment:
`con` is not used, let's remove it and `config.json`.

Reply:
Done